api/apiFunctions.ts

```diff
@@ -64,18 +65,49 @@ export async function logoutAccount(): Promise<{}> {
   }
 }
 
+/**
+ * Get the current user
+ *
+ * @return {Models.DocumentList<Models.Document> | Error} - The user object or an error
+ */
+export async function getCurrentUser(userId: IUser['id']): Promise<IUser> {
+  try {
+    const user = await databases.listDocuments(
+      appwriteConfig.databaseId,
+      Collection.USERS,
+      [Query.equal('userId', userId)],
+    );
+
+    return {
+      id: user.documents[0].userId,
+      email: user.documents[0].email,
+      league: user.documents[0].league,
+    };
+  } catch (error) {
+    console.error(error);
+    throw new Error('Error getting current user');
+  }
+}
+
 /**
  * Get all NFL teams
- * @returns {Models.DocumentList<Models.Document> | Error} - The list of NFL teams
+ *
+ * @return {INFLTeam | Error} - The list of NFL teams
  */
-export const getNFLTeams = async (): Promise<Models.Document[]> => {
+export const getNFLTeams = async (): Promise<INFLTeam[]> => {
   try {
     const response = await databases.listDocuments(
       appwriteConfig.databaseId,
       Collection.NFL_TEAMS,
     );
 
-    return response.documents;
+    const nflTeams = response.documents.map((team) => ({
+      $id: team.$id,
```

Comment:
This comes in as $id, but we don't need to save it as $id. I'd prefer this to be `teamId`

Reply:
✅ fixed